src/config.ts

```diff
@@ -146,6 +150,11 @@ export const init = async () => {
   // Prepare local setting to avoid complicated checks later
   settings.opencast.local = settings.opencast.local && settings.opencast.url === window.location.origin;
 
+  // Prevent malicious callback urls
+  settings.callbackUrl = settings.allowedCallbackPrefixes.some(
```

Comment:
Here it is called `allowedCallbackPrefixes` in the configuration file and earlier in the code, it is `allowedCallbackDomains`. Since your example contains the URL schema, I think that prefix makes more sense.

But in any case, I think this will not work like this. You have to use one or the other.

Not that I've tested this :D

Reply:
Thanks for pointing it out. Originally, I called this setting `allowedCallbackDomains` and later I renamed it to `allowedCallbackPrefixes`. Unfortunately, I missed this line.

Reply:
Now this option should be named persistently `allowedCallbackPrefixes`.